content/plugins/anselmh/kirby-uniform-turnstile/plugin.txt

```diff
@@ -0,0 +1,32 @@
+Title: Turnstile for Uniform
+
+----
+
+Repository: https://github.com/anselmh/kirby-uniform-turnstile
+
+----
+
+Category: forms
+
+----
+
+Subcategory: utilities
+
+----
+
+Description: A plugin for the Uniform plugin implementing the Cloudflare Turnstile protection (sort of a captcha-successor).
+
+----
+
+Example:
+
+```html+php
+<form action="<?= $page->url() ?>" method="post">
+    <!-- ... -->
+
+    <?= csrf_field() ?>
+    <?= turnstileField() ?>
+    <input type="submit" value="Submit">
+</form>
+<?= turnstileScript() ?>
+```
```

Comment:
This file should be in `uniform-turnstile` directory instead of `kirby-uniform-turnstile` to stable work for plugin update check. Because your plugin registry is `anselmh/uniform-turnstile`: https://github.com/anselmh/kirby-uniform-turnstile/blob/main/index.php#L7

Reply:
Thanks, you’re right! Fixed it now.